packages/python/plotly/plotly/express/_core.py

```diff
@@ -1007,6 +1007,99 @@ def build_dataframe(args, attrables, array_attrables):
     return args
 
 
+def _discrete_agg(x):
+    if len(x) == 1:
+        return x.iloc[0]
+    else:
+        return ""
+
+
+def process_dataframe_hierarchy(args):
+    """
+    Build dataframe for sunburst or treemap when the path argument is provided.
+    """
+    df = args["data_frame"]
+    path = args["path"][::-1]
+
+    # ------------ Define aggregation functions --------------------------------
+    lambda_discrete = _discrete_agg
```

Comment:
I would just `def lambda_discrete(x):` here instead of above with `_` and then assigning here

Reply:
ok